alacritty.yml

```diff
@@ -480,62 +481,118 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
+#       Enable or Disable Using Vi key bindings
 #   - SearchForward
+#       Start a forward buffer search
 #   - SearchBackward
+#       Start a backward buffer search
 #   - Copy
+#       Copy the selection to the clipboard
 #   - Paste
+#       Paste from the system clipboard
 #   - PasteSelection
+#       Paste from the selection buffer
 #   - IncreaseFontSize
+#       Increase font size
 #   - DecreaseFontSize
+#       Decrease font size
 #   - ResetFontSize
+#       Reset font size to config value
 #   - ScrollPageUp
+#       Scroll Exactly One Page Up
 #   - ScrollPageDown
+#       Scroll Exactly One Page Down
 #   - ScrollHalfPageUp
+#       Scroll a half page up
 #   - ScrollHalfPageDown
+#       Scroll a half page down
 #   - ScrollLineUp
+#       Scroll one line up
 #   - ScrollLineDown
+#       Scroll one line down
 #   - ScrollToTop
+#       Scroll all the way to the top
 #   - ScrollToBottom
+#       Scroll all the way to the bottom
 #   - ClearHistory
+#       Clear the display buffer(s) to remove history
 #   - Hide
+#       Hide the Alacritty window
 #   - Minimize
+#       Minimize the Alacritty window
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
+#       Toggle full screen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty
 #   - ClearLogNotice
+#       Clear warning and error notices
 #   - ClearSelection
+#       Clear active selection
 #   - ReceiveChar
+#       Allow receiving character input
 #   - None
+#       No action
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Attempt to open the element under the cursor, like a URL. See also url.launcher.
 #   - Up
+#       Move one character up
 #   - Down
+#       Move one character down
 #   - Left
+#       Move one character left
 #   - Right
+#       Move one character right
 #   - First
+#       Move to the start of line
 #   - Last
+#       Move to the end of line
 #   - FirstOccupied
+#       Move to the first non-empty cell
```

Comment:
These bindings are actually a bit more complicated than they might seem at first. They first move to the specified character in the current line and then move to the first character across wrapped lines. This is quite a lot of detail and I'm not sure if we want to document it, but I think it might be useful to know about?

Reply:
I'm updating `First` and `Last` with phrasing like _Move to the start of line, including wrapped lines_

Sinc `FirstOccupied` doesn't clearly imply that it is a line-based action, I think we can leave off the note about line wrapping there.

Reply:
> Sinc FirstOccupied doesn't clearly imply that it is a line-based action, I think we can leave off the note about line wrapping there.

It still is though. All these actions move to the first instance in the line first and then move to the first instance across wrapped lines when already at the first instance within the line. They do not move across wrapped lines immediately.